client/signup/steps/woocommerce-install/transfer.tsx

```diff
@@ -26,6 +108,9 @@ export default function Transfer( { goToStep }: Props ): ReactElement | null {
 			</div>
 			<div className="woocommerce-install__content">
 				<div>
+					{ error.transferFailed && 'error...' }
+					{ error.transferFailed && error.transferStatus }
```

Comment:
that's WIP, right?  😅

Reply:
Yes, WIP, needs some work.

Actually managed to trigger this somehow, I like that the progress bar looks like it keeps attempting :smile: 

https://user-images.githubusercontent.com/811776/141056789-1362f4c2-dfa6-4c78-80e4-e0237fb02e2c.mp4